source/developers-guide/plugin-system/index.md

```diff
@@ -405,6 +405,51 @@ class Shopware_Controllers_Frontend_MyController extends \Enlight_Controller_Act
 }
 ```
 
+### Controller registartion using namespaces
+
+The controller registartion with namespaces is available in Shopware 5.3.2 or above.
+
+Bootstrap:
```

Comment:
we try to prevent using the word bootstrap for the base class in the new plugin system. it's confusing, because in the old system, the base class is always called Boostrap

Just write "plugin base class" or something like that